### ISSUE TITLE
select_advanced: Selects automatically the first option instead of using the defined placeholder

### ISSUE THREAD

**Issue by owiekindisch** [rilwis/meta-box#889]
Setup:
```
array(
	'id' => 'my_select',
	'type' => 'select_advanced',
	'placeholder' => 'Select'
);
```
The error lays in `inc/fields/select.php`. If I c/p the code of the previous version, it works.

**Comment by funkedgeek** [comment]
I'm using the latest version from the Github and the placeholder is working fine.  Please update from the Github version, because I think we fixed this in a previous commit

**Comment by rilwis** [comment]
We fixed it in #870
